autosubmitconfigparser/config/configcommon.py

```diff
@@ -1837,12 +1838,31 @@ def reload(self, force_load=False, only_experiment_data=False, save=False):
             self.experiment_data = self.deep_read_loops(self.experiment_data)
             self.experiment_data = self.substitute_dynamic_variables(self.experiment_data)
             self.experiment_data = self.substitute_dynamic_variables(self.experiment_data, in_the_end=True)
+            if not self.experiment_data.get("AUTOSUBMIT", {}):  # Reserved namespace for autosubmit
```

Comment:
Or to avoid the extra dict, just `if 'AUTOSUBMIT' not in self.experiment_data:` (i.e. check if the key is missing then add it, instead of return a default dict if empty, and confirm the dict is empty).